### PR TITLE
Add tags to events

### PR DESCRIPTION
**Pull request by ricardovaleriano**
TODO:

  - [x] filter events list by tags
  - [x] add some specs
  - [x] tags on events page

### CODE REVIEW

**Comment by lucianosousa** [comment]
why do not merge the rails 4.2 first? :)

**Comment by ricardovaleriano** [comment]
This is the idea Lucas.
Note that this is a branch based on the one that have the changes to move
to rails 4.2.

**Comment by lucianosousa** [comment]
@tauil just merged master in this branch. we can continue working on this now ;)

**Comment by tauil** [comment]
:+1: great!

**Comment by lucasrenan** [comment]
@ricardovaleriano are you still working on this?

**Comment by lucianosousa** [comment]
@lucasrenan I am, but feel free to continue. I'm quite busy these days

**Comment by lucianosousa** [comment]
@lucasrenan I'm back here. feel free to continue on other issues.

@tauil can you just take a look on events page here? I think we can improve the layout of these tags. :)

**Comment by lucianosousa** [comment]
screen shot 2016-08-27 at 9 23 37 pm

**Comment by tauil** [comment]
The tags are separated by commas or spaces? if spaces I would suggest to `parameterize` them.

**Comment by tauil** [comment]
According to what I tested, the tags are being created as one, even if I separate them by commas.

**Comment by tauil** [comment]
Actually, it's right. I've created a event with tags separated by space instead of comma. :+1:

File: app/views/events/tags.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<%= render 'events', events: @events %>
```
Comment (jonasporto) [inline app/views/events/tags.html.erb]: Hey, what you think about use `<%= render @events %> `to avoid be a little redundant?

Comment (jonasporto) [inline app/views/events/tags.html.erb]: For sure!! I was wrong, sorry for that! :+1:

**Pull request merged**